targets/csharp/source/source/Uunit/tests/PlayFabApiTest.cs

```diff
@@ -537,10 +480,169 @@ private void GetObjects2Continued(PlayFabResult<GetObjectsResponse> result, UUni
+            };
+
+            var loginTask = PlayFabClientAPI.LoginWithCustomIDAsync(loginRequest, null, testTitleData.extraHeaders).Result;
+            var loginTask2 = PlayFabClientAPI.LoginWithCustomIDAsync(loginRequest2, null, testTitleData.extraHeaders).Result;
+            var loginTask3 = PlayFabClientAPI.LoginWithCustomIDAsync(loginRequest3, null, testTitleData.extraHeaders).Result;
+
+            testContext.NotNull(loginTask.Result, "Login Result is null for loginRequest");
+            testContext.NotNull(loginTask2.Result, "Login Result is null for loginRequest2");
+            testContext.NotNull(loginTask3.Result, "Login Result is null for loginRequest3");
+            testContext.IsNull(loginTask.Error, "Login error occured for loginRequest: " + loginTask.Error?.ErrorMessage ?? string.Empty);
+            testContext.IsNull(loginTask2.Error, "Login error occured for loginRequest2: " + loginTask2.Error?.ErrorMessage ?? string.Empty);
+            testContext.IsNull(loginTask3.Error, "Login error occured for loginRequest3: " + loginTask3.Error?.ErrorMessage ?? string.Empty);
+            testContext.NotNull(loginTask.Result.AuthenticationContext, "AuthenticationContext is not set for loginRequest");
+            testContext.NotNull(loginTask2.Result.AuthenticationContext, "AuthenticationContext is not set for loginRequest2");
+            testContext.NotNull(loginTask3.Result.AuthenticationContext, "AuthenticationContext is not set for loginRequest3");
+
+            var task = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task2 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask2.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task3 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask3.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task4 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask3.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task5 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask2.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task6 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task7 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask2.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
+            var task8 = PlayFabClientAPI.GetUserDataAsync(new GetUserDataRequest() { AuthenticationContext = loginTask3.Result.AuthenticationContext }, null, testTitleData.extraHeaders);
```

Comment:
3. You're creating 8 tasks to load the same data repeatedly, which isn't really going to properly test the isolation of those logins. This part makes me think I should go look at the original test, because there's probably a design error with the test. For now, reduce it to just 3 calls, 1 for each [separate] player.